src/transform.ts

```diff
@@ -207,6 +208,35 @@ export interface WindowTransform {
   sort?: SortField[];
 }
 
+export interface JoinAggregateDef {
+  /**
+   * The aggregation operation to apply (e.g., sum, average or count). See the list of all supported operations [here](https://vega.github.io/vega-lite/docs/aggregate.html#ops).
+   */
+  op: AggregateOp;
+
+  /**
+   * The data field for which to compute the aggregate function. This can be omitted for window functions that do not operate over a field such as `count`.
```

Comment:
>  for window functions

This no longer "window functions" right?

Reply:
Fixed